src/services/reporting/messageLeads.ts

```diff
@@ -0,0 +1,21 @@
+import toQueryString from "../../lib/toQueryString"
+import { Paginated, UserMessageLead } from "../../index"
+import { ApiCallOptions, fetchPath } from "../../base"
+
+export const fetchMessageLeads = async ({
+  offset,
+  page,
+  size,
+  options = {},
+}: {
+  offset?: number
+  page?: number
+  size?: number
+  options?: ApiCallOptions
+} = {}): Promise<Paginated<UserMessageLead>> => {
+  const query = toQueryString({ offset, page, size })
+  return fetchPath({
+    path: `users/me/message-leads${query ? "?" + query : ""}`,
```

Comment:
```suggestion
    path: `users/me/message-leads${query ? `?${query}` : ""}`,
```

Reply:
what happens when `query` is an empty object? :)

Reply:
Query is always a string so if it's empty nothing will be appended to the URL

Reply:
oh, it runs through `toQueryString` above - thanks! I thought to have identified it as an object in the types, but I probably didn't check properly and assumed too much